haddock-library/src/Documentation/Haddock/Parser.hs

```diff
@@ -848,9 +849,13 @@ parseValid = p some
 -- 'String' from the string it deems valid.
 identifier :: Parser (DocH mod Identifier)
 identifier = do
+  ns <- asum [ Value <$ Parsec.char 'v'
+             , Type <$ Parsec.char 't'
+             , pure None
+             ]
   o <- idDelim
   vid <- parseValid
   e <- idDelim
-  return $ DocIdentifier (o, vid, e)
+  return $ DocIdentifier (Identifier ns o vid e)
   where
-    idDelim = Parsec.satisfy (\c -> c == '\'' || c == '`')
+    idDelim = Parsec.oneOf "\'`"
```

Comment:
Hm. I still don't think this change is a good idea. If we really want to use `oneOf` here, there's no need to escape the single tick.

Reply:
The only reason this is still `satisfy` is because `attoparsec` doesn't have a `oneOf` and I forgot to switch this particular occurrence in the `attoparsec` -> `parsec` PR.

Eeking out this sort of performance in the `haddock-library` parser is pointless - parsing is a pretty trivial part of Haddock's overall runtime.